### PR TITLE
Awesome API Documentation

### PR DESCRIPTION
**Pull request by dignifiedquire**
Ref https://github.com/ipfs/js-ipfs/issues/615

Preview: https://multiformats.github.io/js-multibase/

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,3 +1,141 @@
...
+ * @returns {undefined}
+ */
+function validEncode (name, buf) {
+  const base = getBase(name)
+  base.decode(buf.toString())
+}
+
+/**
+ * @param {string|number} nameOrCode
+ * @returns {Base}
```
Comment (VictorBjelkholm) [inline src/index.js]: Where is `Base` defined?

Comment (dignifiedquire) [inline src/index.js]: In `base.js`, but currently not documented as it is private

Comment (VictorBjelkholm) [inline src/index.js]: Hm, so people get the idea that a `Base` is returned, but they have no idea of what a base is. We should document at least what the base includes here, to not confuse people.

Comment (dignifiedquire) [inline src/index.js]: removed for now

**Pull request merged**